compiler/lookups.nim

```diff
@@ -169,7 +169,8 @@ proc ensureNoMissingOrUnusedSymbols(c: PContext; scope: PScope) =
             getSymRepr(c.config, s))
       inc missingImpls
     elif {sfUsed, sfExported} * s.flags == {}:
-      if s.kind notin {skForVar, skParam, skMethod, skUnknown, skGenericParam}:
```

Comment:
Why not simply list ``skEnumField`` here unconditionally? Every enum field is used since it affects its ordinal number.

Reply:
ok, done